test/operator-framework-e2e/setup.sh

```diff
@@ -0,0 +1,193 @@
+#! /bin/bash
+
+set -e
+
+#################################################################
+# setup.sh is used to build operators using the operator-sdk and
+# build the image + bundle image, and create a FBC image for the
+# following bundle formats:
+# - registry+v1
+# - plain+v0
+# This script will ensure that all images built are loaded onto
+# a KinD cluster with the name specified in the arguments.
+# The following environment variables areused for configuring this script:
+# - $REGISTRY_CTRL_IMG - the tag for the registry+v1 operator image. 
+#     Defaults to "oc-opdev-e2e.operatorframework.io/registry:v0.0.1"
+# - $REGISTRY_BUNDLE_IMG - the tag for the registry+v1 operator bundle image.
+#     Defaults to "oc-opdev-e2e.operatorframework.io/registry-bundle:v0.0.1"
+# - $PLAIN_CTRL_IMG - the tag for the plain+v0 operator image.
+#     Defaults to "oc-opdev-e2e.operatorframework.io/registry:v0.0.1"
+# - $PLAIN_BUNDLE_IMG - the tag for the plain+v0 operator bundle image.
+#     Defaults to "oc-opdev-e2e.operatorframework.io/plain-bundle:v0.0.1"
+# - $CATALOG_IMG - the tag for the catalog image that contains the plain+v0 and registry+v0 operator bundle.
+#     Defaults to "oc-opdev-e2e.operatorframework.io/catalog:e2e"
+# setup.sh also takes 4 arguments. Usage:
+# setup.sh [OPERATOR_SDK] [CONTAINER_RUNTIME] [KUSTOMIZE] [KIND] [KIND_CLUSTER_NAME]
```

Comment:
It would be nice to detect when the user has not passed the correct number of args and print this

Reply:
Done in https://github.com/operator-framework/operator-controller/pull/503/commits/cc277fcd2703a5aaab68e7802480fc8b6fc7412b - includes validation on required environment variables